nais/nais-prod.yaml

```diff
@@ -129,3 +129,6 @@ spec:
       value: "https://isnarmesteleder.intern.nav.no"
     - name: ISNARMESTELEDER_CLIENT_ID
       value: "prod-gcp.teamsykefravr.isnarmesteleder"
+    - name: SPRING_PROFILES_ACTIVE
+      value: remote
```

Comment:
Hva gjør den remote profilen?

Reply:
Det ser ut som den sørger for at kafka-listenere ikke starter opp lokalt. Det er ikke egentlig noe jeg har lagt til, men flyttet fra Dockerfile.